react/public/dummy_labs.json

```diff
@@ -0,0 +1,10 @@
+[
+  {
+    "name": "Lab 0",
+    "endpoint": "http://lab0.stager.ca"
+  },
+  {
+    "name": "Lab 1",
+    "endpoint": "http://lab1.stager.ca"
+  }
+]
```

Comment:
The parent domain here is real. Please don't use it. Use our real one, localhost, or example domains that are guaranteed to not be in use.

Reply:
What do you think in terms of a plan for rolling out this code? If we merge it in while there is still just one STAGER instance (CHEO), will `labs.json` exist? Will `dummy_labs.json` be committed to source? And then once we have the CCM deployment, is `labs.json` the file that will have the ccm-specific endpoint configuration?

Reply:
I put the `dummy_labs.json` for it to be committed to the source so that we have a reference json. As for deployment for single endpoints, we could either not have the json or have it be an empty array, the app behavior would be identical. The app treats missing json as `[]`.